src/com/google/enterprise/adaptor/CustomFormatter.java

```diff
@@ -65,6 +65,24 @@ public CustomFormatter() {
     }
   }
 
+  private CustomFormatter(boolean useColor) {
+    setUseColor(useColor);
+  }
+
+  /** A CustomFormatter that forces the use of colors. */
+  public static class Color extends CustomFormatter {
+    public Color() {
+      super(true);
```

Comment:
This isn't what I meant, though it's what I wrote, and I like this better. I just meant ignore the useColor property, rather than bypassing the auto-detect as well. But this is basically Color/NoColor is a full replacement for useColor, while the base CustomFormatter implements the auto-detect, and is not deprecated. SGTM.